### PR TITLE
Remove unnecessary feeds

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by mmitche** [comment]
dotnet7-transport is required because of https://github.com/dotnet/roslyn/commit/7dc47902defd703f59148843aadcfb6e8fc4012f. @mavasani I'm not sure why that particular version was chosen, or whether it could be changed.

**Comment by mavasani** [comment]
> dotnet7-transport is required because of [7dc4790](https://github.com/dotnet/roslyn/commit/7dc47902defd703f59148843aadcfb6e8fc4012f). @mavasani I'm not sure why that particular version was chosen, or whether it could be changed.

No specific reason. It was probably the latest version built out of main branch of roslyn-analyzers repo that was available at that time. We can definitely bump it to the latest available version as of now.

**Comment by mmitche** [comment]
> > dotnet7-transport is required because of [7dc4790](https://github.com/dotnet/roslyn/commit/7dc47902defd703f59148843aadcfb6e8fc4012f). @mavasani I'm not sure why that particular version was chosen, or whether it could be changed.
> 
> No specific reason. It was probably the latest version built out of main branch of roslyn-analyzers repo that was available at that time. We can definitely bump it to the latest available version as of now.

Fantastic. I'll do so.

File: eng/Versions.props [hunk 1]
```diff
@@ -95,7 +95,7 @@
     <MicrosoftCodeAnalysisVisualBasicCodeRefactoringTestingXUnitVersion>$(MicrosoftCodeAnalysisTestingVersion)</MicrosoftCodeAnalysisVisualBasicCodeRefactoringTestingXUnitVersion>
     <MicrosoftCodeAnalysisVisualBasicCodeStyleVersion>$(CodeStyleAnalyzerVersion)</MicrosoftCodeAnalysisVisualBasicCodeStyleVersion>
     <MicrosoftCodeAnalysisAnalyzerUtilitiesVersion>3.3.0</MicrosoftCodeAnalysisAnalyzerUtilitiesVersion>
-    <MicrosoftCodeAnalysisPerformanceSensitiveAnalyzersVersion>3.3.4-beta1.22504.1</MicrosoftCodeAnalysisPerformanceSensitiveAnalyzersVersion>
+    <MicrosoftCodeAnalysisPerformanceSensitiveAnalyzersVersion>3.11.0-beta1.23464.2</MicrosoftCodeAnalysisPerformanceSensitiveAnalyzersVersion>
```
Comment (mmitche) [inline eng/Versions.props:98]: Updated to latest on 8.0.1xx channel.

File: eng/Versions.props [hunk 2]
```diff
@@ -37,7 +37,7 @@
          but not higher than our minimum dogfoodable Visual Studio version, or else
          the generators we build would load on the command line but not load in IDEs. -->
     <SourceGeneratorMicrosoftCodeAnalysisVersion>4.1.0</SourceGeneratorMicrosoftCodeAnalysisVersion>
-    <MicrosoftILVerificationVersion>7.0.0-alpha.1.22060.1</MicrosoftILVerificationVersion>
+    <MicrosoftILVerificationVersion>7.0.0-preview.1.22076.8</MicrosoftILVerificationVersion>
```
Comment (mmitche) [inline eng/Versions.props:40]: ILVerification changed some interfaces after preview1 and roslyn will need to react.

File: src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj [hunk 3]
```diff
@@ -38,17 +38,23 @@
...
   </PropertyGroup>
+  
+  <ItemGroup>
+    <!--
+        Set the minimum runtime to a .NET 7 prerelease so that prerelease SDKs will be considered during rollForward.
+        RollForward values for roslyn are set in eng/config/runtimeconfig.template.json
+    -->
+    <FrameworkReference Update="Microsoft.NETCore.App"
+                        RuntimeFrameworkVersion="7.0.0-preview.7.22375.6" />
+    <FrameworkReference Update="Microsoft.WindowsDesktop.App"
```
Comment (mmitche) [inline src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj:55]: This doesn't appear to work as expected at the moment.

Comment (mmitche) [reply src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj:55]: The runtimeframeworkversion is set appropriately for M.N.A but not for M.WD.A

**Pull request closed without merging**